google/earthenterprise
# Handle portable globe symlinks in rpm upgrade

Fix for #1187.  Temporarily moves a symlink to external globes directory (if it exists) to avoid having the RPM extraction changing the ownership and permissions of the symlink target.

Verification steps:

- Verify clean installation of PR branch (no upgrade scenario) is successful for both geserver and gefusion
- Verify upgrade from a previous version of opengee with symlinks is ok

1. Install a previous version of opengee such as 5.2.5
1. Replace `/opt/google/gehttpd/htdocs/cutter/globes` with a symlink to `/my/test/dir/globes`
1. Create and set the ownership on `/my/test/dir/globes` for the current user
1. Build RPMs from this PR branch and install opengee-common, opengee-server, and opengee-fusion
1. Verify after upgrading that the ownership and permissions on `/my/test/dir/globes` remain unchanged and that `/opt/google/gehttpd/htdocs/cutter/globes` is still a symlink to that directory
